src/fparser/two/Fortran2008.py

```diff
@@ -83,13 +83,18 @@
     SeparatorBase,
     Type_Declaration_StmtBase,
     StmtBase,
+    NoMatchError,
```

Comment:
It's not used in this file. Is there a reason to import it here?

Reply:
I removed it and nothing broke so apparently not :-)